src/ui/handler/scroll_zoom.js

```diff
@@ -79,15 +79,15 @@ class ScrollZoomHandler {
 
     /**
      * Set the zoom rate of a trackpad
-     * @param {number} [zoomRate = 1/100]
+     * @param {number} [zoomRate=1/100] Zoom rate in some unit.
      */
     setZoomRate(zoomRate: number) {
         this._defaultZoomRate = zoomRate;
     }
 
     /**
      * Set the zoom rate of a mouse wheel
-     * @param {number} [wheelZoomRate = 1/450]
+     * @param {number} [wheelZoomRate=1/450] Zoom rate in some unit.
```

Comment:
nit: It might be misleading to see that type of information, I'd rather leave it out than abstract, but that's personal preference.

Reply:
@karimnaaji oh yeah, I left this in intentionally because I have no idea what the unit is :) I was hoping folks from the GL JS team would help fill in the gaps!

Reply:
The way this is used is intended to be a scalar applied per frame tick to the delta value of a scroll wheel event, which can be in a few different units like DOM_DELTA_LINE, or DOM_DELTA_PIXELS.

Maybe we can speak of _how_ it influences the behavior and link to the different potential event units:
```
A scaling factor applied per frame tick to the delta of a WheelEvent in WheelEvent.deltaMode unit, higher is faster
```
and refer to https://developer.mozilla.org/en-US/docs/Web/API/WheelEvent/deltaMode or other.